browser/src/VariantList/ExportVariantsButton.tsx

```diff
@@ -335,8 +356,31 @@ export type VariantTableVariant = {
   }
 }
 
+type FilteredAlleleFrequency = {
+  popmax: number | null
+  popmax_population: string | null
+}
+
+type V2VariantTableVariant = VariantTableVariant & {
+  exome?: {
+    faf95: FilteredAlleleFrequency
+  }
+  genome?: {
```

Comment:
Is this coming direct from GraphQL? If so should be `| null` rather than `?`.

Reply:
Aha, yep right you are, it is indeed coming directly from GraphQL. Added a fixup commit that changes that.